src/Middleware/HttpLogging/test/FileLoggerProcessorTests.cs

```diff
@@ -46,7 +46,7 @@ await using (var logger = new FileLoggerProcessor(new OptionsWrapperMonitor<W3CL
                     // Pause for a bit before disposing so logger can finish logging
                     try
                     {
-                        await WaitForFile(fileName).DefaultTimeout();
+                        await WaitForFile(fileName, 11).DefaultTimeout();
```

Comment:
Not a huge fan of magic numbers, even if it's a test. Could we extract the message into a variable then string.Length it instead? That way someone doesn't accidentally change the message without updating this value in the future.

And do this for all the tests if you think it's a good idea.